app/models/dhs1171_pdf.rb

```diff
@@ -0,0 +1,38 @@
+class Dhs1171Pdf
+  FIELDS = [
+    "applying_for_food_assistance",
+    "birth_day",
+    "birth_month",
+    "birth_year",
+    "city",
+    "county",
+    "full_name",
+    "signature",
+    "signature_date",
+    "state",
+    "street_address",
+    "zip",
+  ].freeze
+
+  def initialize(client_info)
+    @source_pdf = "DHS_1171.pdf"
+    @client_info = client_info
+  end
+
+  def save(new_pdf)
```

Comment:
Yes, it is a new file name! Good call on changing, I hadnt really paid attention to that because I was doing a lot of copy-pasting from Joel's SAR 7 work.

Renamed at 63474ed